### ISSUE TITLE
Feature request: network send/receive 

### ISSUE THREAD

**Issue by petermcculloch** [stretta/BEAP#52]
Would be cool to have a network module for sending/receiving BEAP data to/from other computers.  Might focus just on sending triggers first since those are simplest (and if they go missing, the worst that happens is that they're missing, as opposed to a gate which never shuts off...)

Thinking that this might be an interesting and simple way of introducing network performance.

**Comment by stretta** [comment]
Yeah, I'm having a collaborative moment over here as well. I tried sending real time MIDI over bluetooth, HAHA NOPE. Check out the Beap score protocol in the MIDI quantizer module - there are some scale lookup functions and the ability to set a list of notes - that sort of thing. Might also be good to think about in the context of networking.

**Comment by unriginal** [comment]
I did a quick thing that takes in gates and triggers, converts them into messages, sends them over a network, and then changes the messages back into a trigger or gate. I'm traveling right now, so I have no way of testing it over several computers on a network, but I think performance is going be poor just based off of conversion from and to signal. I know it's possible to send and receive buffers reliably, so it could be possible to share wavetables (I think sending the scale key would work better than sending the buffer for a score or quantizer though).
